### PR TITLE
Setup Google Analytics

### PR DESCRIPTION
**Pull request by andriyParashchuk**
We have GA installed: https://github.com/jetthoughts/jetthoughts.github.io/tree/archive-20180327

need to set up on new site the same ID

### CODE REVIEW

**Comment by pftg** [comment]
@andriyParashchuk any updates on this issue? what our next actions?

**Comment by pftg** [comment]
any updates?

**Comment by andriyParashchuk** [comment]
@pftg I'm not sure I understand. I added Google Analytics in this branch and master (deploy) branch

**Comment by pftg** [comment]
what we are going to do with this issue? do you need any help verficiation? do we need to close it if it's verified?

**Comment by pftg** [comment]
do not leave open issue without status on what stage of development this issue is.

File: _includes/analytics.html [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
   (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-  ga('create', 'UA-10749382-8', 'auto');
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+  ga('create', 'UA-10749382-1', 'jetthoughts.com');
```
Comment (pftg) [inline _includes/analytics.html:6]:
```js
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-10749382-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-10749382-1');
</script>
```

**Comment by andriyParashchuk** [comment]
@pftg provide access for Google Analytics

**Pull request closed without merging**